pkg/matcher/annotation_matcher.go

```diff
@@ -201,7 +201,7 @@ func MatchPipelinerunByAnnotation(ctx context.Context, logger *zap.SugaredLogger
 				continue
 			}
 			if out != types.True {
-				logger.Warnf("CEL expression is not matching %s, skipping", prun.GetGenerateName())
+				logger.Debugf("CEL expression for PipelineRun %s is not matching, skipping", prun.GetName())
```

Comment:
I think we should not make it as Debug :thinking: because we need to know this info by default when annotation doesn't match isn't it  I mean should show this log the way its showing today without changing any configuration in CM

Reply:
okay sounds good

Reply:
added some more code into the commit to get properly the nane of the prun